agents/helpers/agent.py

```diff
@@ -341,6 +343,31 @@ def __init__(self, graph, goal, agent_id):
         # Create initial path to goal
         self.compute_shortest_path()
 
+    def old_transition_cost(self, from_node, to_node):
```

Comment:
What makes this transition cost different than the one we use now?

Reply:
We left this one in as a backup, but it can indeed be removed!